source/index.html.md

```diff
@@ -630,6 +630,27 @@ object][ApplePay-PaymentToken] for more information.
 </p>
 
 
+##### Method: `samsungpay`
+
+Samsung Pay provides the payment details as either JWE or JWE/JWS JOSE objects.
+Currently only the JWE is accepted.
+
+<dl class="dl-vertical">
+  <dt>samsungpay[jwt]</dt>
+  <dd>
+    <a href="https://tools.ietf.org/html/rfc7516#section-3.1"><span class="type">Compact serialized JWE token</span></a>
+    <br>
+    Complete JWE token as received from Samsung Pay.
+  </dd>
+  <dt>samsungpay[cek]</dt>
+  <dd>
+    <span class="type">[:base64:]</span>
+    <br/>
+    16 byte symmetric AES content encryption key (CEK) used to encrypt the
```

Comment:
```suggestion
    The (symmetric AES) Content Encryption Key (CEK) used to encrypt the
```
I guess nobody would need to know about the number of bytes? I suspect it's only relevant to document the length of the Base64 encoded key (which is always 24 if I am not mistaken)?

Reply:
Well, I chose to write 16 bytes because the algorithm is 128bit AES, this was just to emphasize what key size we're talking about. I'm trying to use the text to give hints about what to expect.

Though I do think it should be changed to 128 bits.